src/schema/v2/partner/updatePartnerFlagsMutation.ts

```diff
@@ -0,0 +1,90 @@
+import {
```

Comment:
I might rename this mutation, as it's not generically updating partner flags (and accepting partner flags as inputs). It's specifically updating a partner flag which is currently reflecting the 'update last CMS access timestamp'.

So should the mutation just do that? As in, rather than `udatePartnerFlagsMutation`, updateCMSLastAccessTimestamp` or something? Just to keep things explicit and specific.